subjects/localhost/README.md

```diff
@@ -0,0 +1,162 @@
+## Localhost
+
+The Hypertext Transfer Protocol (HTTP) is an application layer protocol in the Internet protocol suite model for distributed, collaborative, hypermedia information systems.
+HTTP is the foundation of data communication for the World Wide Web. Hypertext documents include hyperlinks to other resources that the user can easily access, for example, by a mouse click or tap on the screen in a web browser.
+
+### Instructions
+
+#### Server
+
+The project can be written in one of these languages [Go,C++,C...]
```

Comment:
If Rust is allowed it should be written explicitly since it is part of the common cursus (C is not), and actually created for this kind of usage (Go is not really suited for servers). The order should then be Rust, Go, C++, C (Go before C++ just because C++ is not part of the cursus).

Reply:
I Do Agree with your order (We will apply it). One more thing Go is suited for an HTTP server if you google it:.`
Go is ideal for creating cloud service applications thanks to its scalability, concurrency, and ability to deliver idiomatic APIs. These features are why Google uses Go as the primary language for its cloud computing services. Other popular cloud platforms that use Go include Docker, Dropbox, and Terraform.`

reference(https://distantjob.com/blog/golang-web-development/)